roles/contrail_node/tasks/main.yaml

```diff
@@ -0,0 +1,55 @@
+---
+- when:
+  - docker_registry is not defined
+
+  block:
+  - name: Untar Contrail tar-file
+    unarchive:
+      src: "{{ contrail_docker_images_path }}/contrail-kubernetes-docker-images_{{ contrail_version }}.tgz"
+      dest: "/tmp"
+
+  - name: Load Contrail-vRouter-Agent image
+    docker_image:
+      name: contrail-agent
+      tag: "{{ contrail_version }}"
+      timeout: 300
+      load_path: "/tmp/contrail-agent-{{ contrail_os_release }}-{{ contrail_version }}.tar.gz"
+
+  - name: Load Contrail-Kubernetes-Agent image
+    docker_image:
+      name: contrail-k8s-agent
+      tag: "{{ contrail_version }}"
+      timeout: 300
+      load_path: "/tmp/contrail-kubernetes-agent-{{ contrail_os_release }}-{{ contrail_version }}.tar.gz"
+
+- when:
+  - inventory_hostname == contrail_first_master
+
+  block:
+  - name: Add contrail service account to privileged scc
+    oc_adm_policy_user:
+      user: system:serviceaccount:kube-system:contrail
+      resource_kind: scc
+      resource_name: privileged
+      state: present
+    delegate_to: "{{ contrail_first_master }}"
+
+  - name: Add the daemonset-controller service account to privileged scc
```

Comment:
Are there going to be only two service accounts? If the list will grow I would consider using the `with_items` construction.